### PR TITLE
frontend/qt: gitignore libbreez_sdk_bindings.so

### PR DESCRIPTION
**Pull request by Beerosagos**

### CODE REVIEW

File: frontends/qt/.gitignore [hunk 1]
```diff
@@ -2,3 +2,4 @@
 /config/
 /server/libserver.h
 /server/libserver.so
+/server/libbreez_sdk_bindings.so
```
Comment (thisconnect) [inline frontends/qt/.gitignore:5]: I also have `/server/libbreez_sdk_bindings.dylib` that could maybe be ignored?

Comment (Beerosagos) [reply frontends/qt/.gitignore:5]: Looks like it's still in the Makefile. I added it too

**Review by benma — COMMENTED** [review]
NACK

This change is wrong, it seems there was a rebase error of Ross' PR onto my PRs that fixed the linking of breez.

These files are not supposed to be in the server/ folder at all, and they are not copied there in my PRs:

https://github.com/digitalbitbox/bitbox-wallet-app/pull/2378/files
https://github.com/digitalbitbox/bitbox-wallet-app/pull/2377/files
https://github.com/digitalbitbox/bitbox-wallet-app/pull/2373/files

Yet on current staging-ln, they are copied there

I thought I had checked Ross' PR for this and it was correct at some point, but the one that was merged has this wrong copies again:

https://github.com/digitalbitbox/bitbox-wallet-app/pull/2528/files#diff-caac7b534206f54ce0fbaa69fc4f8c4472c3da4a29fe7fc737c4f8dbc5a34166

**Comment by benma** [comment]
The BitBox.pro changes are also wrong:

https://github.com/digitalbitbox/bitbox-wallet-app/pull/2528/files#diff-babf163ef2b81ff8385eb3040eefeac2f6451545b2408a630c2b6e29e586a48b

**Comment by benma** [comment]
PR to fix the rebase error: https://github.com/digitalbitbox/bitbox-wallet-app/pull/2593

**Comment by Beerosagos** [comment]
@benma sorry for not catching the error reviewing Ross's PR! I approved #2593 and now close this. Thanks :pray:

**Pull request closed without merging**